h3i/src/main.rs

```diff
@@ -93,14 +93,15 @@ fn config_from_clap() -> std::result::Result<Config, String> {
         .arg(
             Arg::with_name("host:port")
                 .help("Hostname and port of the HTTP/3 server")
-                .required(true)
+                .required(false)
```

Comment:
This should stay required. If we want to omit the SNI, then add a new `--no-sni` option, which means you can avoid changing the prompter code and then just pass None to `quiche::connect` if we don't want one